### PR TITLE
bpo-46724: Use `JUMP_ABSOLUTE` for all backward jumps.

### PR DESCRIPTION
**Pull request by markshannon**
https://bugs.python.org/issue46724

### CODE REVIEW

File: Python/compile.c [hunk 1]
```diff
@@ -7530,6 +7530,11 @@ normalize_jumps(struct assembler *a)
                 last->i_opcode = JUMP_FORWARD;
             }
         }
+        if (last->i_opcode == JUMP_FORWARD) {
+            if (last->i_target->b_visited) {
```
Comment (iritkatriel) [inline Python/compile.c]: A few rows above there is "if (last->i_target->b_visited == 0)"  so to be consistent you could check "== 1" here.

**Review by iritkatriel — APPROVED** [review]

**Comment by iritkatriel** [comment]
It will be interesting to repeat the stats after this.

File: Python/compile.c (hunk 1 above)
Comment (markshannon) [reply Python/compile.c]: Testing against zero is always quicker (at least never slower), so I'll change it to `!= 0`.

Comment (markshannon) [reply Python/compile.c]: Except it's a one bit value, so the compiler can work that out for itself. `== 1` it is then.

**Pull request merged**